extmod/modussl_mbedtls.c

```diff
@@ -184,12 +186,14 @@ STATIC mp_obj_ssl_socket_t *socket_new(mp_obj_t sock, struct ssl_args *args) {
         assert(ret == 0);
     }
 
+    if (args->do_handshake.u_bool) {
     while ((ret = mbedtls_ssl_handshake(&o->ssl)) != 0) {
```

Comment:
I assume you want to keep the diff smaller, but I think you should indent here.

Reply:
> I assume you want to keep the diff smaller, but I think you should indent here.

Yeah, this comes from my branch, where I indeed try to minimize deltas to avoid conflicts. Fixed.